bugbug/bug_features.py

```diff
@@ -505,6 +505,14 @@ def __call__(self, bugs, **kwargs):
         )
 
 
+class couple_delta_creation_date(couple_bug_feature):
+    def __call__(self, bugs, **kwargs):
+        delta = datetime.strptime(bugs[0], "%Y-%m-%dT%H:%M:%SZ") - datetime.strptime(
```

Comment:
It'd be better to use `dateutil.parser`, so we'd be more robust to possible changes in the format.